apps/server/test/modules/files-storage/file-storage-with-s3-server.e2e-spec.ts

```diff
@@ -207,7 +207,6 @@ describe('file-storage controller (e2e)', () => {
 			it('should return status 404 for file not found', async () => {
 				const response = await api.getDownloadFile(`/file/download/${validId}/wrong-name.txt`);
 
-				expect(response.error.message).toEqual(`The requested FileRecord: ${validId} has not been found.`);
```

Comment:
We should figure out why validId is pass as Bson Object [Object object].

Reply:
https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/files-storage/files-storage.module.ts#L75

Here is a problem that the 'where' variable is supposed to be in JSON.stringify(where)

Reply:
note as todo for later